packages/xarc-app-dev/src/config/opt2/webpack-options.ts

```diff
@@ -55,11 +55,23 @@ export type WebpackOptions = {
   devArtifactsPath?: string;
 
   /**
-   * Set to true to explicitly enable CSS module support for all style files
-   * - **Default: `undefined` (auto detect)**
-   * - If not set, then check env `CSS_MODULE_SUPPORT`
+   * Configure CSS module support.
+   *
+   * Settings:
+   * 1. `true` or "cssOnly" (**default**): CSS module for any file with `.css` extension
+   * 2. RegExp: CSS module for any file that matches the custom RegExp
+   * 3. `"all"`: CSS module for all style files (`css`, `styl`, `sass`, `scss`)
+   * 4. `"byModExt"`: Use the preset RegExp that matches any file ending with `.mod.{ext}` or `.module.{ext}`
+   *    ie: `/\.(mod|module)\.(css|styl|sass|scss)$/i`
+   * 5. `false`: Disable CSS module completely
+   *
+   *
+   * Note: CSS module support require the package `@xarc/opt-postcss`, please add it to your `devDependencies`.
```

Comment:
Now`@xarc/opt-postcss` is not mandatory for css modules. I have removed that check in previous commits.

https://github.com/electrode-io/electrode/pull/1751/files#diff-9fcd63a47bc3b00243ba5c324ad645a1d43f54402bab17ee1e5f2dc8ad239834L61